cijoe/scripts/plotter.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python
+"""
+Helper functions for plotting benchmark graphs
+==============================================
+
+
+
+.. note::
+    This uses matplotlib and numpy for plotting
+"""
+import logging as log
+from pathlib import Path
+from typing import Dict, List
+
+import matplotlib.pyplot as plt
+from cijoe.core.resources import dict_from_yamlfile
+
+OUTPUT_NORMALIZED_FILENAME = "benchmark-output-normalized.json"
+PLOT_SCALE = 1.5
```

Comment:
It seems that two scripts import this, but the original value is not 1.5 for both. `latency_plotter` originally had 1.3.
Is this a problem?

Reply:
No, it only defines how much "air" there is above the bars in the plot. 1.5 looks fine, in my opinion